Nos/Service/Relay/RelayService.swift

```diff
@@ -210,7 +210,6 @@ extension RelayService {
         let contactFilter = Filter(
             authorKeys: [authorKey],
             kinds: [.contactList],
-            limit: 1,
```

Comment:
Let's revert this change, see my other comment.

Reply:
Done!